frontend/canvas-chrome-ext/src/scripts/instructor-inject-script.jsx

```diff
@@ -244,13 +325,40 @@ function initTerminalFrame() {
     return terminalFrame;
 }
 
-function initInstructorViewContainer() {
-    let instructorViewContainer = document.createElement("div");
-    instructorViewContainer.id = "instructor-view-container";
-    instructorViewContainer.style.margin = "20px";
-    instructorViewContainer.style.height = "100%";
-    instructorViewContainer.style.overflowY = "scroll";
-    return instructorViewContainer;
+//APP BAR SPECIFIC FUNCTIONS
+function InstructorView_appBar(){
```

Comment:
could you follow camelCase naming convention here to match the rest of the functions